operators/community-windows-machine-config-operator/3.1.0/manifests/community-windows-machine-config-operator.3.1.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,327 @@
+          - use
+        serviceAccountName: windows-machine-config-operator
+      deployments:
+      - name: windows-machine-config-operator
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              name: windows-machine-config-operator
+          strategy: {}
+          template:
+            metadata:
+              labels:
+                name: windows-machine-config-operator
+            spec:
+              containers:
+              - args:
+                - --debugLogging
+                command:
+                - windows-machine-config-operator
+                env:
+                - name: WATCH_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.annotations['olm.targetNamespaces']
+                - name: POD_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.name
+                - name: OPERATOR_NAME
+                  value: community-windows-machine-config-operator
+                image: quay.io/openshift-windows/community-windows-machine-config-operator:community-4.8-a8b33fb
+                imagePullPolicy: IfNotPresent
+                name: manager
+                resources: {}
+              hostNetwork: true
+              nodeSelector:
+                node-role.kubernetes.io/master: ""
+              priorityClassName: system-cluster-critical
+              serviceAccountName: windows-machine-config-operator
+              terminationGracePeriodSeconds: 10
+              tolerations:
+              - effect: NoSchedule
+                key: node-role.kubernetes.io/master
+                operator: Exists
+              - effect: NoExecute
+                key: node.kubernetes.io/unreachable
+                operator: Exists
+                tolerationSeconds: 120
+              - effect: NoExecute
+                key: node.kubernetes.io/not-ready
+                operator: Exists
+                tolerationSeconds: 120
+      permissions:
+      - rules:
+        - apiGroups:
+          - ""
+          - coordination.k8s.io
+          resources:
+          - configmaps
+          - leases
+          verbs:
+          - get
+          - list
+          - watch
+          - create
+          - update
+          - patch
+          - delete
+        - apiGroups:
+          - ""
+          resources:
+          - events
+          verbs:
+          - create
+          - patch
+        serviceAccountName: windows-machine-config-operator
+    strategy: deployment
+  installModes:
+  - supported: true
+    type: OwnNamespace
+  - supported: false
+    type: SingleNamespace
+  - supported: false
+    type: MultiNamespace
+  - supported: false
+    type: AllNamespaces
+  keywords:
+  - windows
+  maintainers:
+    - email: team-winc@redhat.com
+      name: Red Hat, Windows Container Support for OpenShift
+  maturity: preview
+  minKubeVersion: 1.21.0
```

Comment:
Shouldn't be 1.22.0?

Reply:
Nope, this is targeting 4.8 which is k8s 1.21